languages/ruby/stack.yaml

```diff
@@ -0,0 +1,8 @@
+flags: {}
+extra-package-dbs: []
+packages:
+- '.'
+- '../../'
+extra-deps:
+- c-storable-deriving-0.1.3
```

Comment:
I _think_ we shouldn’t need this extra-dep because we don’t depend on c-storable-deriving in the languages.

Reply:
It looks like we need it since the language packages have `haskell-tree-sitter` as a build target.